python/lsst/pipe/base/pipeline_graph/_dataset_types.py

```diff
@@ -193,29 +196,31 @@ def name(self) -> str:
     @property
     def key(self) -> NodeKey:
         """Key that identifies this dataset type in internal and exported
-        networkx graphs.
+        networkx graphs (`~lsst.pipe.base.pipeline_graph.NodeKey`).
         """
         return NodeKey(NodeType.DATASET_TYPE, self.dataset_type.name)
 
     @property
     def dimensions(self) -> DimensionGroup:
-        """Dimensions of the dataset type."""
+        """Dimensions of the dataset type (`~lsst.daf.butler.DimensionGroup`)."""
```

Comment:
This line seems to be a catch-22 with Ruff. This version complains that the line is too long, but changing to ```
```suggestion
        """Dimensions of the dataset type (`~lsst.daf.butler.DimensionGroup`).
        """
```
causes it to complain that it's misformatted.